metabrainz/templates/macros.html

```diff
@@ -9,7 +9,7 @@
     {% set alert_class="alert-info" %}
   {% endif %}
 
-  <div class="alert {{ alert_class }}">{{ message }}</div>
+  <div class="alert {{ alert_class }}">{{ _(message) }}</div>
```

Comment:
Alright, I just realised that this message will obviously be passed to the macro, which itself is probably wrapped around gettext.

Will change this one.